.github/workflows/test-all.yaml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - main
+      - add-dockerfile
```

Comment:
Ahah! I'm still seeking a good github workflow (especially the Github Actions) and am not up to speed on all the "cool kid" tricks :-) 

How would I use the `pr-test-all` when I'm testing my own local branch?

Reply:
It works on PRs — so we'd need to open a PR, then add that label. I don't think it will work without a PR (though maybe you could open one locally on your fork I guess)

Reply:
I removed it - it'll get called when my PR next hits